nsinit/config.go

```diff
@@ -70,6 +71,22 @@ func modify(config *configs.Config, context *cli.Context) {
 	config.AppArmorProfile = context.String("apparmor-profile")
 	config.ProcessLabel = context.String("process-label")
 	config.MountLabel = context.String("mount-label")
+
+	userns_uid := context.Int("userns-root-uid")
+	maxUid := 1 << 32
```

Comment:
Ah, well 2^32 is not a valid integer (int32). The largest unsigned integer is 2^32 -1. Should this be max unsigned int or max signed int? (both int32)